Lib/glyphsLib/builder/builders.py

```diff
@@ -134,7 +133,10 @@ def __init__(
         # Map Glyphs layer IDs to UFO layer names.
         self._layer_map = {}
 
-        # List of exploded color layer when building minimal UFOs.
+        # List of exploded color palette layers when building minimal UFOs.
+        self._color_palette_layers = []
```

Comment:
I find the name "color palette layers" a bit confusing. I think you mean here the solid-colors-only kind of CORLv0, how about `self._solid_color_layers`?

Reply:
I can see how this can be confusing. The name is derive from the name of Glyphs 3 layer attribute. Color palette layers specify a palette index, when the other color layers set the colors directly on the path.